src/development/coding-secrets-detection.md

```diff
@@ -143,3 +145,6 @@ type  = "file"
 target = "gitleaks.toml"
 value = "gitleaks.toml"
 ```
+
+### Ignore false positives
```

Comment:
Is content intended to follow this heading?

Reply:
Thanks, well caught.
Fixed